src/Microsoft.DotNet.Build.Tasks.Packaging/src/GetApplicableAssetsFromPackages.cs

```diff
@@ -109,25 +106,44 @@ public override bool Execute()
                 Log.LogError($"Could not locate compile assets for any of the frameworks {String.Join(";", compileFxs.Select(fx => fx.ToString()))}");
             }
 
+            List<ITaskItem> candidateRuntimeAssets = new List<ITaskItem>();
+
             foreach (var runtimeFx in runtimeFxs)
             {
-                RuntimeAssets = _resolver.ResolveRuntimeAssets(runtimeFx, TargetRuntime)
+
+                candidateRuntimeAssets = _resolver.ResolveRuntimeAssets(runtimeFx, TargetRuntime)
                                          .Where(ra => !NuGetAssetResolver.IsPlaceholder(ra))
                                          .Select(ra => PackageItemAsResolvedAsset(_targetPathToPackageItem[ra]))
-                                         .ToArray();
+                                         .ToList();
 
-                if (RuntimeAssets.Any())
+                if (candidateRuntimeAssets.Any())
                 {
-                    Log.LogMessage($"Resolved runtime assets from {runtimeFx.ToString()}: {String.Join(";", RuntimeAssets.Select(r => r.ItemSpec))}");
+                    Log.LogMessage($"Resolved runtime assets from {runtimeFx.ToString()}: {String.Join(";", candidateRuntimeAssets.Select(r => r.ItemSpec))}");
                     break;
                 }
             }
 
-            if (!RuntimeAssets.Any())
+
+            if (!candidateRuntimeAssets.Any())
             {
                 Log.LogError($"Could not locate runtime assets for any of the frameworks {String.Join(";", runtimeFxs.Select(fx => fx.ToString()))}");
             }
+            else
+            {
 
+                ITaskItem[] matchingSymbols = (from asset in candidateRuntimeAssets
+                                               where File.Exists(Path.ChangeExtension(asset.ItemSpec, ".pdb")) 
```

Comment:
Not sure if ItemSpec is right here.  I had a different idea of how to do this.  Mind if I take over the fix?